job-dsls/jobs/deploy_jobs.groovy

```diff
@@ -194,7 +194,10 @@ for (repoConfig in REPO_CONFIGS) {
             }
         }
 
-        if (repo == "optaplanner") {
+        if (repo == "optaplanner" ||
+            repo == "optaweb-employee-rostering" ||
+            repo == "optaweb-vehicle-routing" ||
+            repo == "optaplanner-quickstarts") {
```

Comment:
All of these use release=8, but "release" is only available in JDK 11+ (not in JDK 8). So they need to be build with JDK 11, even though they are JDK 8 compatible.